src/main/java/edu/kis/vh/nursery/defaultCountingOutRhymer.java

```diff
@@ -1,34 +1,73 @@
-package edu.kis.vh.nursery;
+public class DefaultCountingOutRhymer {
+    int size = 12;
+    int err_code = -1;
+    private int[] numbers = new int[size];
+    private int total = err_code;  
 
-public class defaultCountingOutRhymer {
+    // Getter dla pola total
+    public int getTotal() {
+        return total;
+    }
 
-    private int[] NUMBERS = new int[12];
+//<<<<<<< Accesory
+    // Getter i setter dla numbers (w razie potrzeby)
+    public int[] getNumbers() {
+        return numbers.clone();  
+    }
 
-    public int total = -1;
+    public void setNumbers(int[] numbers) {
+        if (numbers.length == size) {
+            this.numbers = numbers.clone();
+        }
+    }
+//=======
+    private static final int NUM = 12;
+    private static final int TOTAL = -1;
+    private static final int FULL = 11;
+
+    private final int[] numbers = new int[NUM];
+    public int total = TOTAL;
+//>>>>>>> master
 
     public void countIn(int in) {
-        if (!isFull())
-            NUMBERS[++total] = in;
+        if (!isFull()) {
+            numbers[++total] = in;
+        }
+    }
+
+    
+
+    public boolean callCheck() {
+//<<<<<<< Accesory
+        return total == err_code;
+=======
+        return total == TOTAL;
+//>>>>>>> master
+    }
+
+    public boolean isFull() {
+        return total == FULL;
     }
 
-        public boolean callCheck() {
-            return total == -1;
+    protected int peekaboo() {
+        if (callCheck()) {
+//<<<<<<< Accesory
+            return err_code;
+//=======
+            return TOTAL;
+//>>>>>>> master
         }
```

Comment:
Conflicts not resolved. Code does not compile I cannot check it, but extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).